src/main/java/at/hannibal2/skyhanni/features/garden/visitor/VisitorListener.kt

```diff
@@ -41,6 +44,12 @@ class VisitorListener {
     private var lastClickedNpc = 0
     private val logger = LorenzLogger("garden/visitors/listener")
 
+    private val patternGroup = RepoPattern.group("garden.visitor")
+    private val tablistExistPattern by patternGroup.pattern(
+        "tablist.enabled",
+        "§b§lVisitors:.*"
+    )
+
```

Comment:
instead of this why not just use `VisitorApi.visitorCountPattern.matches`
which is: `"§b§lVisitors: §r§f\\((?<info>.*)\\)"`

Reply:
good catch!